src/Markup/Perspex.Markup.Xaml/PerspexXamlLoader.cs

```diff
@@ -187,3 +187,15 @@ private static IEnumerable<Uri> GetUrisFor(Type type)
         }
     }
 }
+
+namespace Perspex
+{
+	public static class XamlObjectExtensions
+	{
+		public static ObjectT LoadFromXaml<ObjectT>(this ObjectT obj)
```

Comment:
Can we use `TObject` for the generic type? https://msdn.microsoft.com/en-us/library/0zk36dx2.aspx